tests/test_spec.py

```diff
@@ -409,3 +412,96 @@ def test_apispec_serve_spec_preserve_order(self, app):
         response_json_docs = client.get("/api-docs/openapi.json")
         assert response_json_docs.status_code == 200
         assert response_json_docs.json["paths"] == paths
+
+
+@pytest.fixture
+def flask_cli_runner(app):
+    return app.test_cli_runner()
+
+
+@pytest.fixture
+def temp_file(tmp_path):
+    file_path = tmp_path / "openapi_file"
+    yield str(file_path)
+    with contextlib.suppress(FileNotFoundError):
+        file_path.unlink()
+
```

Comment:
My point was that you could remove the fixture and use the pytest fixture directly in each test.

```py
def test_apispec_command_write(tmp_path):
    result = flask_cli_runner.invoke(args=cmd.split() + [tmp_path / "openapi_file"])
```

Reply:
I don't like, when tests leave side effects after the execution. With fixture it's much easier to control it.